### PR TITLE
CU-2hbpk7y: Fix RN tooltips error.

### PR DESCRIPTION
**Pull request by haly88**

### CODE REVIEW

File: packages/react-ameliorate-component-tooltip/source/tooltip.js [hunk 1]
```diff
@@ -33,12 +33,16 @@ export const Tooltip = componentFactory('Tooltip', ({ Parent, componentName }) =
...
+        //###if(!MOBILE) {###//
+        caption =  ('' + caption).split(/[\r\n]/g).reduce((result, line, i, a) => {
           result.push(line);
           if (i < a.length)
             result.push(<br key={i}/>);
           return result;
         }, []);
+        //###}###//
+
+        return caption;
```
Comment (jpodwys) [inline packages/react-ameliorate-component-tooltip/source/tooltip.js:45]: What issue is this fixing?

Comment (haly88) [reply packages/react-ameliorate-component-tooltip/source/tooltip.js:45]: @jpodwys @cthatcher-ev `<br` is not supported on React Native, so pushing a tooltip on RN will crash the APP.

Comment (utterboredom) [reply packages/react-ameliorate-component-tooltip/source/tooltip.js:45]: Does this affect iPhone safari at all?  If it does you would want `!RN` instead so its only altering Native.

Comment (haly88) [reply packages/react-ameliorate-component-tooltip/source/tooltip.js:45]: @utterboredom It does not, only Native APP.

Comment (utterboredom) [reply packages/react-ameliorate-component-tooltip/source/tooltip.js:45]: in that case I would change your !MOBILE to !RN

Comment (utterboredom) [reply packages/react-ameliorate-component-tooltip/source/tooltip.js:45]: Ignore me - the RN option is apparently only in our eVisit-RN repo - not RA

**Review by utterboredom — APPROVED** [review]

**Comment by marcomafessolli** [comment]
@haly88 @cthatcher-ev maybe this could be an option to fix the issue? https://github.com/ds300/patch-package

**Pull request closed without merging**